### PR TITLE
feat(contextselector): add actions footer

### LINKED ISSUE #3475: Add actions footer to the Context menu
Add an optional footer to the Context menu component. The footer can hold a button right aligned in the footer according to this design.

image

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #3475

### CODE REVIEW

**Review by mcoker — APPROVED** [review]
nice!

**Review by mcarrano — APPROVED** [review]
Looks good to me @christiemolloy .  @gdoyle1 can you also take a look?

File: src/patternfly/components/ContextSelector/examples/context-selector.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #ws-core-c-context-selector-basic {
   min-height: 380px;
 }
+
+#ws-core-c-context-selector-with-footer {
+  min-height: 380px;
+}
```
Comment (mattnolting) [inline src/patternfly/components/ContextSelector/examples/context-selector.css:7]:
```suggestion
}

```

**Review by mattnolting — APPROVED** [review]
LGTM!

**Review by gdoyle1 — APPROVED** [review]
@christiemolloy looks awesome! Thank you

**Pull request merged**